### PR TITLE
Basket item component #31

### PR DESCRIPTION
**Pull request by matthall8**
Updates: 

- Created HTML and CSS for basket component #31 
- Refactored Meal Card for use elsewhere
- Updated Footer CSS styling
- Added global h1 styling

### CODE REVIEW

File: client/src/Router.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export default class Router extends Component {
         <Switch>
           <Route exact path="/" component={MealsListPage} />
```
Comment (jema28) [inline client/src/Router.js]: I've noticed you're adding a header and footer to each page. You can place these before and after your Switch.

Comment (matthall8) [reply client/src/Router.js]: Added header to the Router file, footer has not been added as we only want it on some places.

**Comment by matthall8** [comment]
@MynahMarie - thanks Mynah. I've solved that in that latest commit 👌

**Comment by DenisKent** [comment]
closing due to development methodology

**Comment by matthall8** [comment]
Reopening as per discussions with QA team.

**Pull request merged**